### PR TITLE
July newsletter

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

File: docs/blog/newsletter-july-2020.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
+.. post:: July 05, 2020
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – July 2020
+#####################################
+
+Hello, everyone! It's Beth here, with the latest edition of the Write the Docs newsletter. As I write to you from here in Switzerland, summer seems to have finally arrived. Here's to bright sun and blue skies ahead, and hopefully a bit of a break as the summer holidays cme along.
```
Comment (plaindocs) [inline docs/blog/newsletter-july-2020.rst]:
```suggestion
Hello, everyone! It's Beth here, with the latest edition of the Write the Docs newsletter. As I write to you from here in Switzerland, summer seems to have finally arrived. Here's to bright sun and blue skies ahead, and hopefully a bit of a break as the summer holidays come along.
```

File: docs/blog/newsletter-july-2020.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+- Cons: many writers just want to write, and there can be less time for that; at some point, you may have to shoulder the burden of firing; plus these `17 Reasons Not to Be a Manager <https://charity.wtf/2019/09/08/reasons-not-to-be-a-manager>`_.
+- Check if you're doing management work already without realising it. Perhaps you take a special joy in managing and planning projects, including making sure your teammates feel good about projects and their work. Or, maybe you're helping with team morale, eliminating barriers to people getting work done, or supporting your peers' career growth.
+- Taking on management responsibilities without being a lead/senior can change how you're perceived, so it's worth being thoughtful about it. `This piece by Tanja Reilly <https://noidea.dog/glue>`_ describes how such so-called "glue work" can affect your career.
+
+-------------------------------
+Gathering documentation feedback
+-------------------------------
+
+To improve documentation, we need to know where it's lacking, and we can get this feedback from a variety of sources, including user interviews and internal surveys. Here's some advice from our community on how to go about it!
```
Comment (plaindocs) [inline docs/blog/newsletter-july-2020.rst]: This first sentence feels a little choppy. I don't have a great suggestion for improvement though.

**Review by plaindocs — APPROVED** [review]
Mere quibbles below.
Great stuff.

File: docs/blog/newsletter-july-2020.rst (hunk 2 above)
Comment (ericholscher) [reply docs/blog/newsletter-july-2020.rst]: Perhaps:
```suggestion
To improve documentation, we need to know where it's lacking. We can get this feedback from a variety of sources, including user interviews and internal surveys. Here's some advice from our community on how to go about it!
```

Comment (plaindocs) [reply docs/blog/newsletter-july-2020.rst]: Solid choice.

**Pull request merged**